common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -991,21 +991,23 @@ legion_doomhorn = {
 }	
 serathil = {
 	max_amount = 1
-	spawn_date = 605.6.1
+	spawn_date = -1.1.1
 	
 	spawn_chance = {
 		value = 100
-		modifier = {
-			factor = 0
-			NOT = { year = 605 }
-		}
 	}
 	
 	weight = {
 		value = 0
 		additive_modifier = {
 			value = 100
 			character = 52058						# Kor'Gall Stonemaul
+			year = 584
```

Comment:
After the year 584, both characters have the same chance to get Serathil. `year = 584` means equal or greater than the year 584.

Reply:
Meme said that as well, but Kash'drakor dies before the next start date